core/src/main/java/auber/com/mygdx/Auber.java

```diff
@@ -1,10 +1,10 @@
-package com.mygdx.auber;
+package auber.com.mygdx;
```

Comment:
I see these everywhere-why?  This file is in com/mygdx/auber, which means its package should be com.mygdx.auber, NOT auber.com.mygdx-in fact, it doesn't even run on my computer because of this!

Reply:
Ah. I might have gotten things a little mixed up when moving the files. It runs on mine!

Reply:
Ah, yeah-may also have been my version of the project.